cli/cache/http_cache/local.rs

```diff
@@ -932,7 +1043,6 @@ mod test {
         json!({
           "modules": {
             "https://deno.land/x/different_content_type.ts": {
-              "path": "deno.land/x/#different_content_ty_f15dc.js",
```

Comment:
This property was also removed because it's unnecessary. We can derive it from the url and headers.